src/main/webapp/resources/js/pages/analysis/components/AnalysisSamples.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useContext, useEffect, useState } from "react";
+import { Card, Row, Icon, Button } from "antd";
+import { AnalysisDetailsContext } from '../../../contexts/AnalysisDetailsContext';
+import { getI18N } from "../../../utilities/i18n-utilties";
+
+import {
+    getAnalysisInputFiles,
+    getForwardFile,
+    getReverseFile
+} from "../../../apis/analysis/analysis";
+
+export default function AnalysisSamples() {
+  const { analysisDetailsContext, loadAnalysisSamples } = useContext(AnalysisDetailsContext);
+
+  useEffect(() => {
+    loadAnalysisSamples();
+  }, []);
+
+  const downloadReferenceFile = () => {
+    if (analysisDetailsContext.referenceFile.identifier !== undefined)
+    {
+        window.open(`${window.TL.BASE_URL}referenceFiles/download/${analysisDetailsContext.referenceFile.identifier}`, '_blank');
+    }
+  };
+
+  const renderReferenceFile = () => {
+    const referenceFile = [];
+
+    if(analysisDetailsContext.referenceFile.length == 0)
```

Comment:
Make sure your code is formatted with Prettier rules.  Open braces on new lines can cause code errors in javascript.

Reply:
Fixed in 4d5b63fdf69f55254f7b8ec40e149f8fc26fcd4a